scripts/tools/a2server-aliases.txt

```diff
@@ -17,17 +17,16 @@ alias rasppleii-update='raspbian-update a2cloud a2server'
 
 alias welcome-message-edit='sudo nano /etc/motd'
 
-alias showip='ifconfig eth0 | grep "inet\ addr" | cut -d: -f2 | cut -d" " -f1'
-alias showmac='ifconfig eth0 | grep "HWaddr" | cut -dH -f2 | cut -c7-23'
-alias showip-wifi='ifconfig wlan0 | grep "inet\ addr" | cut -d: -f2 | cut -d" " -f1'
-alias showmac-wifi='ifconfig wlan0 | grep "HWaddr" | cut -dH -f2 | cut -c7-23'
-alias ifreset='sudo rm /etc/udev/rules.d/70-persistent-net.rules; echo Interfaces removed. You should system-restart now.'
-
-alias netatalk-stop="sudo /etc/init.d/netatalk stop; bonjour-off"
-alias netatalk-start="sudo /etc/init.d/netatalk start; bonjour-on; grep -q 'ATALKD_RUN=yes' /etc/default/netatalk"
-alias netatalk-restart="sudo /etc/init.d/netatalk restart; bonjour-on; grep -q 'ATALKD_RUN=yes' /etc/default/netatalk"
-alias netatalk-off='sudo /etc/init.d/netatalk stop; bonjour-off; sudo update-rc.d -f netatalk remove &> /dev/null'
-alias netatalk-on='sudo update-rc.d netatalk defaults &> /dev/null; bonjour-on; sudo /etc/init.d/netatalk restart'
+alias showip='ip a show dev enp0s3 | grep "inet " | cut -dt -f2 | cut -db -f1'
+alias showmac='ip a show dev enp0s3 | grep "link/ether" | cut -dr -f2 | cut -db -f1'
+alias showip-wifi='ip a show dev wlan0 | grep "inet " | cut -dt -f2 | cut -db -f1'
+alias showmac-wifi='ip a show dev wlan0 | grep "link/ether" | cut -dr -f2 | cut -db -f1'
+
+alias netatalk-stop="sudo systemctl stop atalkd cnid afpd papd timelord a2boot macipgw; bonjour-off"
+alias netatalk-start="sudo systemctl start atalkd cnid afpd papd timelord a2boot macipgw &> /dev/null; bonjour-on"
+alias netatalk-restart="sudo systemctl restart atalkd cnid afpd papd timelord a2boot macipgw &> /dev/null; bonjour-on"
```

Comment:
The `cnid` service is designed to be started on demand by the `afpd` process. It should not be controlled directly.